clients/java/src/main/java/io/camunda/zeebe/client/api/fetch/FormGetRequest.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.fetch;
+
+import io.camunda.zeebe.client.api.command.FinalCommandStep;
+import io.camunda.zeebe.client.protocol.rest.FormItem;
+
+public interface FormGetRequest extends FinalCommandStep<FormItem> {}
```

Comment:
Instead of returning the protocol's `FormItem`, I think you need to use a DTO 
You should create an interface of the response model, maybe you can extend the existing `io.camunda.zeebe.client.api.response.Form`

Reply:
Well pointed @houssain-barouni, applied the changes on https://github.com/camunda/camunda/pull/22640#:~:text=for%20Form%20client-,ba8a6a5,-Merge%20state